app/src/main/java/cm/aptoide/pt/abtesting/experiments/SimilarAdExperiment.java

```diff
@@ -0,0 +1,47 @@
+package cm.aptoide.pt.abtesting.experiments;
+
+import cm.aptoide.pt.abtesting.ABTestManager;
+import cm.aptoide.pt.app.AdsManager;
+import cm.aptoide.pt.app.ApplicationAdResult;
+import java.util.List;
+import rx.Scheduler;
+import rx.Single;
+
+/**
+ * Created by franciscoaleixo on 08/10/2018.
+ */
+
+public class SimilarAdExperiment {
+  private final String AD_IDENTIFIER = "ASV-1068-AppNext_Test";
```

Comment:
Maybe call this EXPERIMENT_ID like in other experiments? I think it makes more sense;